jsamchineme/send-it
# Ft frontend user signup 162406112

#### What does this PR do?
- Add user signup and login feature
#### Description of Task to be completed?
- Implement Form validations
- Implement Signup and Login 
- Implement service to handle server side errors
- Implement service to handle client side input errors
#### How should this be manually tested?
- pull branch, _npm run start:dev_ and check `http://localhost:3001/login` and `http://localhost:3001/signup` pages
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/162406112
#### Screenshots (if appropriate)
None
#### Questions:`
None
